django_cyverse_auth/models.py

```diff
@@ -224,7 +223,7 @@ def lookupSessionToken(request):
     """
```

Comment:
I don't really see that changing the comment at top of the file nor to token is really necessary.  Just because we don't/cannot reuse tokens, doesn't mean that the class wasn't designed with that in mind.